views/guide.lavender

```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```

Comment:
@mowchan is this space between if and elseif a convention we have/want to implement?